app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="inaka.lewisdemo"
+    android:versionCode="@android:integer/config_longAnimTime">
+
+    <application
```

Comment:
According to **Lewis**:
> ### Message: App is not indexable by Google Search; consider adding at least one Activity with an ACTION-VIEW intent-filler. See issue explanation for more details..
 >### Explanation: .
 >Adds URLs to get your app into the Google index, to get installs and traffic to your app from Google Search.